Source/WebCore/Modules/fetch/FetchBodyOwner.cpp

```diff
@@ -129,7 +129,7 @@ void FetchBodyOwner::blob(Ref<DeferredPromise>&& promise)
     if (isBodyNullOrOpaque()) {
         auto* context = promise->scriptExecutionContext();
         promise->resolveCallbackValueWithNewlyCreated<IDLInterface<Blob>>([this, context](auto&) {
-            return Blob::create(context, Vector<uint8_t> { }, Blob::normalizedContentType(extractMIMETypeFromMediaType(m_contentType)));
```

Comment:
We can probably do some additional refactoring here: remove the context variable and get it from the function.
```
promise->resolveCallbackValueWithNewlyCreated<IDLInterface<Blob>>([this](auto& context) {
```

Reply:
thanks for the tips! I will fix the capturing context in upcoming commits.